http/pool.go

```diff
@@ -138,10 +144,12 @@ func (s *WSPool) RemoveClient(c WSSpeaker) {
 	defer s.Unlock()
 	for i, ic := range s.speakers {
 		if ic.GetHost() == c.GetHost() {
+			logging.GetLogger().Debugf("succeeded RemoveClient %s for pool %s", c.GetHost(), s.GetName())
 			s.speakers = append(s.speakers[:i], s.speakers[i+1:]...)
 			return
 		}
 	}
+	logging.GetLogger().Debugf("failed RemoveClient %s for pool %s", c.GetHost(), s.GetName())
```

Comment:
"Failed to remove client %s from pool %s"

Reply:
fixed